pkg/metrics/metrics.go

```diff
@@ -30,13 +30,22 @@ const (
 	ValueSucceededTrue = "true"
 	// ValueSucceededFalse is value False for metric label failed.
 	ValueSucceededFalse = "false"
+	// ValueRestoreSingleMemberInMN is value for metric of single member restoration in multi-node.
+	ValueRestoreSingleMemberInMN = "single_member"
```

Comment:
I would recommend to use the name `ValueRestoreSingleMemberInMultiNode` instead.